django_admin_dracula/templates/admin/base_site.html

```diff
@@ -1,10 +1,12 @@
-{% extends "admin/base.html" %} {% block title %}{% if subtitle %}{{ subtitle }}
-| {% endif %}{{ title }} | {{ site_title|default:_('Django site admin') }}{%
-endblock %} {% block branding %}
-<div id="site-name">
-  <a href="{% url 'admin:index' %}"
-    >{{ site_header|default:_('Django administration') }}</a
-  >
-</div>
-{% if user.is_anonymous %} {% include "admin/color_theme_toggle.html" %} {%
-endif %} {% endblock %} {% block nav-global %}{% endblock %}
```

Comment:
I reverted the formatting changes here since it added a newline in the `{% endif %}` Django template tag which caused an error in Django's template rendering